File: Assignmentno18/task_1.java
import java.util.*;

import javax.swing.plaf.basic.BasicSliderUI.ScrollListener;
class Logic 
{
    void Prime(int iValue)
    {
        

       int iCnt=0;
       boolean flag =false;
       for(iCnt=2;iCnt<iValue/2;iCnt++)
       {
            if((iValue%iCnt)==0)
            {
                flag = true;
                break;
              
            }   
       }   
       if(flag==true)
       {
        System.out.println(iValue+"The number is prime");

       }
       else
       {
        System.out.println(iValue+"The number is not prime");
       }
        
    }
}

class task_1
{
    public static void main(String A[])
    {
        int iNo=0;
        Scanner sobj = new Scanner(System.in);
        Logic lobj = new Logic();


        System.out.println("Enter the number :");
        iNo=sobj.nextInt();
        lobj.Prime(iNo);


        
    }
}

File: Assignmentno12/task_1.c
#include<stdio.h>

void Print_factors(int iNo)
{
    int iCnt =0;
    for(iCnt=1;iCnt<=iNo/2;iCnt++)
    {
        if((iNo%iCnt)==0)
        {
          printf("%d\t",iCnt);
        }
   }

}

int main()
{
    int number;

    printf("Enter number :\n");
    scanf("%d",&number);

    Print_factors(number);

    return 0;
}